scripts/setup-centos8.sh

```diff
@@ -132,6 +141,38 @@ function install_protobuf {
   )
 }
 
+function install_grpc {
+  wget_and_untar https://github.com/grpc/grpc/archive/refs/tags/v1.48.1.tar.gz grpc
+  (
+    cd grpc
+    cmake_install -DgRPC_BUILD_TESTS=OFF -DgRPC_ABSL_PROVIDER=package -DgRPC_ZLIB_PROVIDER=package -DgRPC_CARES_PROVIDER=package -DgRPC_RE2_PROVIDER=package -DgRPC_SSL_PROVIDER=package -DgRPC_PROTOBUF_PROVIDER=package -DgRPC_INSTALL=ON
+  )
+}
+
+function install_crc32c {
+  wget_and_untar https://github.com/google/crc32c/archive/refs/tags/1.1.2.tar.gz crc32c
+  (
+    cd crc32c
+    cmake_install -DCRC32C_BUILD_TESTS=OFF -DCRC32C_BUILD_BENCHMARKS=OFF -DCRC32C_USE_GLOG=OFF
+  )
+}
+
+function install_nlohmann_json {
+  wget_and_untar https://github.com/nlohmann/json/archive/refs/tags/v3.11.3.tar.gz nlohmann_json
+  (
+    cd nlohmann_json
+    cmake_install -DJSON_BuildTests=OFF
+  )
+}
+
+function install_google_cloud_cpp {
+  wget_and_untar https://github.com/googleapis/google-cloud-cpp/archive/refs/tags/v2.24.0.tar.gz google_cloud_cpp
+  (
+    cd google_cloud_cpp
+    cmake_install -DGOOGLE_CLOUD_CPP_ENABLE_EXAMPLES=OFF -DGOOGLE_CLOUD_CPP_ENABLE=storage,bigquery
+  )
+}
```

Comment:
From my understanding these (+abseil) are all for google cloud/bq and should only be installed in adapters.sh

Reply:
That's true, though since I was not sure whether this script was used to create the container images or not, I had preferred to keep them until someone asked me to remove it, so I will removing these right now.

Reply:
Ah ok, no problem. The adapters container is build on the centos8 container as a base each using the similarly named script :)